docs/index.rst

```diff
@@ -77,6 +80,20 @@ Now you can connect to port 80 of that ip address. Here's an example
     so you might find this ip address changes over time. You can always find out
     what ip is being used with ``docker-machine ip [machine name]``
 
+You may need to use a reliable hostname to access your container server (e.g. for
+Firefox Accounts). You can set one by editing your ``/etc/hosts`` file on your
+native operating system. For example::
+
+    192.168.99.100  olympia.dev
```

Comment:
A downside with this is docker-machine hands out ips based on the order you start machines. 

It's not a problem if you're only using one, but if someone using the env has multiple machines it makes using hostnames a pain.